pwndbg/ui.py

```diff
@@ -73,6 +73,8 @@ def get_window_size(target=sys.stdin):
     rows, cols = get_cmd_window_size()
     if rows is not None and cols is not None:
         return rows, cols
+    elif os.environ.get("PWNDBG_IN_TEST") is not None:
+        return fallback
```

Comment:
random thought: fallback should never change, right? nobody should ever do `os.environ[a]=b`? can we compute it once at the top of the file?

and then probably run the tests with `LINES=x COLUMNS=y ./tests.sh`?

Reply:
I'm not sure if they are updated ever, is this an optimization? But I don't like the idea of having to pass environment variables to `tests.sh`. Running `./tests.sh` should just work.

Reply:
isn't tests.sh launching an external python code or script? ;D if so then it could insert the env :D.

Reply:
But yeah idk how often is get_window_size called, if often then we should probably optimize it if possible, heh

Reply:
anyway good for now

Reply:
I didn't look at the tests.sh script, so you're probably right. We'd have to make sure gdb doesn't overwrite the COLUMNS during startup.